service.subtitles.legendastv/addon.xml

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon
+   id="service.subtitles.legendastv"
+   name="Legendas.TV"
+   version="2.4.4"
+   provider-name="gfjardim">
```

Comment:
when taking over please add yourself as an author after the last one

Reply:
Like `gfjardim,andrebrait` in this case? (comma-separated, no spaces?)

Reply:
Yes pls